src/norm/postgres.nim

```diff
@@ -316,3 +316,28 @@ template transaction*(dbConn; body: untyped): untyped =
 
     raise
 
+
+# One-to-Many Fetching
+proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: var seq[M]) =
+  const foreignKeyFieldName: string = O.getRelatedFieldNameOn(M)
+  const manyTableName: string = M.table()
+  const sqlCondition: string = manyTableName & '.' & foreignKeyFieldName & " = $1"
+
+  dbConn.select(relatedEntries, sqlCondition, oneEntry.id)
```

Comment:
The intent was mostly for clarity. `M.table()` did not feel to me like it told you sufficiently that the tablename you're getting is of the "many" side from the one-to-many relationship. If you want it gone I can throw it out, not a big deal, just thought this way it would be clearer what you're doing and what you're getting, which means less thinking as you read over the code. and the goal of the `sqlCondition` becomes easier to understand.